result/block-scoped-var.js

```diff
@@ -0,0 +1,57 @@
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
+    if (true) {
+        var build = true;
+    }
+
+    console.log(build);
+}
+
+function doIfElse() {
+    if (true) {
+        var build = true;
+    } else {
+        var build = false;
+    }
+}
+
+function doTryCatch() {
```

Comment:
__[development][eslint]__
* Error - 'doTryCatch' is defined but never used (no-unused-vars)